test/functional/p2p_eviction.py

```diff
@@ -48,6 +48,10 @@ def set_test_params(self):
         # 4 by netgroup, 4 that sent us blocks, 4 that sent us transactions and 8 via lowest ping time
         self.extra_args = [['-maxconnections=32']]
 
+    def setup_network(self):
+        self.disable_mocktime()
```

Comment:
why bitcoin has it working? let's add here TODO maybe that it is just a temporary fix?

Reply:
we enable mocktime by default and bitcoin enables it for some tests only (that's also why we have to bump mocktime instead of using `sleep()` in many tests btw)